web/src/components/navbar/NavBarDesktop.tsx

```diff
@@ -91,9 +94,17 @@ export const NavBarLoggedInDesktop = (props: Props): ReactElement => {
             aria-haspopup="true"
             onClick={toggleDropdown}
           >
-            <div className="text-bold text-primary flex flex-align-center">
-              <Icon className="usa-icon--size-4 margin-right-1">account_circle</Icon>
-              <div>{userName}</div>
+            <div
+              className={`text-bold text-${
+                state.isAuthenticated == "TRUE" ? "primary" : "base"
+              } flex flex-align-center`}
+            >
+              <Icon className="usa-icon--size-4 margin-right-1">
+                {state.isAuthenticated == "TRUE" ? "account_circle" : "help"}
+              </Icon>
+              <div>
+                {state.isAuthenticated == "TRUE" ? userName : Config.navigationDefaults.navBarGuestText}
+              </div>
```

Comment:
with this many inline conditionals checking the same thing, might be nicer to write as a function? like

```

const textColor = state.isAuthenticated == "TRUE" ? "text-primary" : "text-base"
// ...(and same for the other variables)

<div
              className={`text-bold ${textColor} flex flex-align-center`}
            >
              <Icon className="usa-icon--size-4 margin-right-1">
                {state.isAuthenticated == "TRUE" ? "account_circle" : "help"}
              </Icon>
              <div>
                {state.isAuthenticated == "TRUE" ? userName : Config.navigationDefaults.navBarGuestText}
              </div>
```